### DISCUSSION TITLE
Two factor SMS authentication method does not list an option with Ukraine phone number

### DISCUSSION THREAD

**Discussion by ghost** [community/community#59936] [New to GitHub] [answered]
![image](https://github.com/community/community/assets/86713661/eb63cb17-bf5c-40b5-9607-ebbbde3dc25e)

Is there's any reason why?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! These discussions may be of interest to you as the contain more information and workarounds: 
https://github.com/orgs/community/discussions/10285 
https://github.com/orgs/community/discussions/14416
